packages/manager/modules/pci/src/projects/project/data-processing/jobs/data-processing-jobs.controller.js

```diff
@@ -0,0 +1,90 @@
+import {
+  isJobRunning,
+  getClassFromStatus,
+  datagridToIcebergFilter,
+  getDataProcessingUiUrl,
+} from '../data-processing.utils';
+import { DATA_PROCESSING_GUIDE_URL } from '../data-processing.constants';
+
+import { getCriteria } from '../../project.utils';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $state,
+    CucCloudMessage,
+    dataProcessingService,
+    ovhManagerRegionService,
+    atInternet,
+  ) {
+    this.$state = $state;
```

Comment:
Do we need to inject $state here ? I don't find any usage of it in the controller

Reply:
Removed :)